virion.yml

```diff
@@ -1,6 +1,8 @@
 name: apibossbar
 version: 0.0.5
 antigen: xenialdan\apibossbar
-api: [3.13]
-php: [7.2]
+api: 3.13
+php:
+  - 7.2
+  - 8.0
```

Comment:
Remove 7.2 und 8.0 and set it to 7.3

Reply:
Are you serious? This PR allows FormAPI to be used under pmmp's experimental php8 support.
it is designed for php7.2, so I, as a non-author, should not update it from 7.2 to 7.3 without checking the code…

Reply:
PHP 8 has more changes as PHP 7.3. You're lost. You open a fucking PR for adding PHP 8 to php needed versions without checking the code. WTF

Reply:
????????

Reply:
> PS: You are not a maintainer or contributor to this repository. You are irrelevant.
> The maintainer's opinion, not yours, takes precedence and does not need to be reviewed.